docs/getting-started/switch-to-composer.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Switch to Composer"
+description: "How to switch your plugin based Timber theme over to the Composer based version."
+menu:
+  main:
+    parent: "getting-started"
+---
+
+With the upcoming release of Timber 2, we will stop releasing Timber as a plugin. We advise everyone to switch to the Composer based install of Timber 1 as soon as possible as a first step towards upgrading to Timber 2.
+
+## Recommended upgrade path
+
+1. Get a local development version of your website up and running.
+2. Check if you are running the latest version of the Timber plugin. If not, update it and check if your website still runs as expected.
+
+4. Install the latest 1.x version of Timber via Composer
+5. Load the Composer autoloader and initialize Timber
+5. Load the composer autoloader and initialize Timber 
+6. Check if your website still runs as expected
+7. Deploy your changes to your live website
+
+We highly recommend doing these steps on a local development version of your website. If you don’t have one yet, you could use [Local by Flywheel](https://localbyflywheel.com/) to get one up and running in a few minutes.
+
+#### 1.1 How to install Composer
+If you don’t have Composer installed yet, you can follow the [official installation guide](https://getcomposer.org/doc/00-intro.md).
+
+
+### 2. Check if you are running the latest version of the Timber plugin
+We want to make sure that you are running the latest version of the Timber plugin before we start the upgrade process. This way we can be sure that any issues that might occur are not caused by an outdated version of the plugin.
+
+### 3. Disable the Timber plugin
+Once you are sure that you are running the latest version of the Timber plugin, you can disable it. This will make sure that the plugin is not loaded anymore and does not interfere with the Composer based version of Timber.
+
+### 4. Install the latest 1.x version of Timber via Composer
+Now that the plugin is disabled, you can install the latest 1.x version of Timber via Composer. You can do this by navigating to your site's active theme folder inside your terminal and then running the following command:
+
+```shell
+composer require timber/timber:^1.0
+```
+
+If Composer gives you the following notice:
+
+```shell
+composer/installers contains a Composer plugin [...] (writes "allow-plugins" to composer.json)
+```
+it is safe to answer `y` to this question and press enter.
+
+
+### 5. Load the Composer autoloader and initialize Timber
+Now that we have Timber installed via Composer, we need to load the Composer autoloader and initialize Timber. You can do this by adding the following code at the top of your **functions.php** file:
+
+```php
+/**
+ * If you are installing Timber as a Composer dependency in your theme, you'll need this block
+ * to load your dependencies and initialize Timber.
```

Comment:
```suggestion
 * to load dependencies and initialize Timber.
```

Reply:
I have redone this codeblock to be more in line with the function.php of the [Timber starter theme v2 functions file](https://github.com/timber/starter-theme/blob/2.x/functions.php)